### ISSUE TITLE
Evaluate mobile experience in manager

### ISSUE THREAD

**Issue by azizaparveen** [Esri/solutions-components#773]
### Describe requirements

Manager is mobile responsive but in portrait mode Tablets and in smaller devices the experience is pretty basic, Tables tools are not available, wondering if we can look into adding the table tools for mobile device use.
We will need design from IA team for this
![image](https://github.com/user-attachments/assets/2972e0b6-02ad-44c7-b58e-d8c0cb717edd)

**Comment by azizaparveen** [comment]
cc @sarahmcdonald7

**Comment by azizaparveen** [comment]
Pushing this to R2, so we get more time to work on it, @sumitzarkar @sarahmcdonald7

**Comment by sumitzarkar** [comment]
@azizaparveen 

- In Figma design, in landscape mode, we are not allowing users to view/edit the feature as per the first screenshot, and in the second, we are showing the view and delete button. Could you please check and confirm this
![Image](https://github.com/user-attachments/assets/9aa5965c-8b14-4fba-a452-04fc8fa71e94)

**Comment by azizaparveen** [comment]
@sumitzarkar right, we are not currently but we would like to allow editing in mobile device.

**Comment by sumitzarkar** [comment]
- As shown in the below screenshot we can show only one row, so should we show view and delete button or not
  ![Image](https://github.com/user-attachments/assets/e8a76249-d6c2-46d4-913f-00ccb3b29dbe)

- Map tools in mobile mode will be difficult to show
 ![Image](https://github.com/user-attachments/assets/9c8a796a-17cf-4b13-b908-decbb5d8770c)

-  in one screen we see cancle button do we need it?

Create feature form in design | Create feature form in desktop
-- | --
![Image](https://github.com/user-attachments/assets/862049e9-6bb1-41d8-b3e0-38520a3d2d23) | ![Image](https://github.com/user-attachments/assets/18dc95ae-71e9-4394-8777-17556f170ce7)

**Comment by azizaparveen** [comment]
@sumitzarkar I reached out to Instant Apps about your questions, 
1) About landscape space issue, I am waiting for their decision
2) Some Instant App uses horizontal toolbar instead of vertical toolbar we can do that, also we will not show zoom button in mobile device map toolbar. 
![Image](https://github.com/user-attachments/assets/d1de95a6-737a-4d38-8579-d8a66227be8f)
3) We don't like to show Create Features button in mobile device at all.

**Comment by azizaparveen** [comment]
For question 1,
We would like to add map and layer switcher to the toolbar in mobile device mode, I think it will also be helpful for portrait mode as well.

![Image](https://github.com/user-attachments/assets/fe3d8130-6f64-4c9c-ba60-015beff2c892)

**Comment by azizaparveen** [comment]
@sumitzarkar @pratiksabale07 , Seeing some issues and  need some enhancements with mobile interface

- Map in split view is showing horizontal toolbar, we wanted to show it in the mobile interface only
- Do not allow geometry editing, we decided not to allow creating new features in mobile, same reason, do not allow geometry editing
- Do not show snapping tools in mobile interface
- Horizontal scrollbar is not working
 ![Image](https://github.com/user-attachments/assets/82783234-ddcc-4b25-8191-2d31a80c8445)
- Map switcher and layer switcher do not show up right away after switching to mobile view
- When switching between desktop and mobile interface, Manager switch back to default map instead of showing current map

**Comment by pratiksabale07** [comment]
@azizaparveen 
## Cause
- This is an enhancement requested by the client aimed at improving the mobile user experience.
___
## Solution
Changes for mobile devices
  - Remove the map and layer picker bar.
  - Hide the zoom buttons.
  - Show the map and layer picker on the right side of the map.
  - On clicking the icon, the shows the popover as follows:-
 
Popover for map picker | Popover for map layer picker
-- | --
![Image](https://github.com/user-attachments/assets/525f4024-4f7d-4f13-996b-f9627921873b) |  ![Image](https://github.com/user-attachments/assets/f8ab5543-ac6d-4e44-8d61-98e16e44f74f)
 
___
## Dev Checklist
Items  | Is required | Status | Comments
-- | -- | -- | --
Required NLS changes? | Yes   |   | 
Required Backward compatibility | No  |   | 
Required 508 | Yes  |   | 
Need Sanitize? | No  |   | 
___
## Impact Analysis Report
Impacted Areas | Comments
-- | --
Have Configuration changes? | No
Have Runtime changes? | Yes
Impact on RTL | Yes
Impacted components | - map card
Impact on runtime | General Changes  ![Image](https://github.com/user-attachments/assets/cf7bf633-68c0-4672-bf87-d17910ef5efc) <ul><li>Tools on the map should be laid out horizontally</li><li>Overflowing tools should go into an ellipsis</li><li>The home icon should be shown in the tools list</li><ul>
|| Changes in mobile devices ![Image](https://github.com/user-attachments/assets/ea6dae0c-eda9-48d5-ba1c-08542b5d2fbf) <ul><li>Zoom buttons should be hidden</li><li>The map and layer picker icon should be shown at right side of the map</li><li>Update the map and feature table upon selection from popovers</li><li>While switching from mobile to desktop or desktop to mobile, selections should be maintained</li><li>For horizontal mobile mode the Map view and the Feature should each occupy half the screen</li></ul>
|| - After selecting a record in the table, the feature count at the delete and view buttons should be shown, but when viewing a record, the feature count should not be shown.

**Comment by azizaparveen** [comment]
@pratiksabale07 Map is not showing in split view in mobile device

![Image](https://github.com/user-attachments/assets/64b92b99-3d0a-40cd-98a4-497b9a5896cd)

**Comment by azizaparveen** [comment]
Looks good in devext, I see layer and map switcher working in mobile view.
